classes/Deposit.inc.php

```diff
@@ -412,4 +412,31 @@ public function setExportDepositError($exportDepositError) {
 	public function getExportDepositError() {
 		return $this->getData('exportDepositError');
 	}
+
+	/**
+	 * Get Displayed status locale string
+	 * @return string
+	 */
+	public function getDisplayedStatus() {
+		if (!empty($this->getExportDepositError())) {
+			$displayedStatus = __('plugins.generic.pln.displayedstatus.error');
+		} else if ($this->getLockssAgreementStatus()) {
+			$displayedStatus = __('plugins.generic.pln.displayedstatus.completed');
+		} else if ($this->getStatus() == 0) {
```

Comment:
Is there a defined constant for this rather than `0`?

Reply:
Fixed